framework/src/engine/legacy/endpoint.ts

```diff
@@ -77,4 +83,24 @@ export class LegacyEndpoint {
 
 		return decodeBlockJSON(await this.storage.getBlockByHeight(height)).block;
 	}
+
+	public async getLegacyBrackets(_context: RequestContext): Promise<LegacyChainBracketInfo[]> {
+		const legacyBracketInfos = [];
```

Comment:
- Do you mean you couldn't find any utility of this endpoint for users or you are saying there is no usage in legacy? coz if its latter then we endpoints usually don't have any usage within sdk in other places, its used directly by users. If its former, then it would be nice to show the bracket info to the user as they can see what they configured and how much they have synced based on last block. Related to test, i'm adding it.
- imo, Array is singular but it holds multiple bracketInfo which can be clearly seen from `LegacyChainBracketInfo[]`. The google search that you made is in the case when you call something as `LegacyChainBracketInfoArray` but instead you call it `LegacyChainBracketInfoArrays`, that is incorrect